overlord/state/change.go

```diff
@@ -477,6 +477,31 @@ func (c *Change) Tasks() []*Task {
 	return c.state.tasksIn(c.taskIDs)
 }
 
+// LaneTasks returns all tasks from given lanes the state change depends on.
+// All tasks are returned if any of the lanes is 0 (the default lane).
+func (c *Change) LaneTasks(lanes []int) []*Task {
```

Comment:
Quick idea about the API, you could use: `LaneTasks(lanes...int) []*Task` here. This makes using the API a bit more natural, i.e. "LaneTasks(1,2)" or "LaneTasks(1)". I.e.:
```
diff --git a/overlord/state/change.go b/overlord/state/change.go
index 9246b1b6e..5b167b952 100644
--- a/overlord/state/change.go
+++ b/overlord/state/change.go
@@ -479,7 +479,7 @@ func (c *Change) Tasks() []*Task {
 
 // LaneTasks returns all tasks from given lanes the state change depends on.
 // All tasks are returned if any of the lanes is 0 (the default lane).
-func (c *Change) LaneTasks(lanes []int) []*Task {
+func (c *Change) LaneTasks(lanes ...int) []*Task {
        laneLookup := make(map[int]bool)
        for _, l := range lanes {
                if l == 0 {
diff --git a/overlord/state/change_test.go b/overlord/state/change_test.go
index b6a2aea3f..2964799b4 100644
--- a/overlord/state/change_test.go
+++ b/overlord/state/change_test.go
@@ -194,18 +194,16 @@ func (cs *changeSuite) TestLaneTasks(c *C) {
                c.Assert(tasks1, DeepEquals, tasks2)
        }
 
-       c.Assert(chg.LaneTasks([]int{}), HasLen, 0)
-
-       tasks := chg.LaneTasks([]int{0})
+       tasks := chg.LaneTasks(0)
        checkTasks(tasks, []*state.Task{t1, t2, t3, t4, t5})
 
-       tasks = chg.LaneTasks([]int{lane1})
+       tasks = chg.LaneTasks(lane1)
        checkTasks(tasks, []*state.Task{t1, t2, t4})
 
-       tasks = chg.LaneTasks([]int{lane2})
+       tasks = chg.LaneTasks(lane2)
        checkTasks(tasks, []*state.Task{t3, t4})
 
-       tasks = chg.LaneTasks([]int{lane1, lane2})
+       tasks = chg.LaneTasks(lane1, lane2)
        checkTasks(tasks, []*state.Task{t1, t2, t3, t4})
 }
 

```